<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>731</width>
    <height>685</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SiBook</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(186, 219, 173);
border-color: rgb(176, 121, 70);</string>
  </property>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>271</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sitka Heading</family>
     <pointsize>24</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(21, 22, 24); 
font: 8pt &quot;Sitka Heading&quot;; font-size:24pt;</string>
   </property>
   <property name="text">
    <string>Changes:</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>310</y>
     <width>571</width>
     <height>348</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLineEdit" name="name_edit">
      <property name="styleSheet">
       <string notr="true">color: rgb(176, 121, 70);font: 8pt &quot;Eras Demi ITC&quot;; font-size:20pt;</string>
      </property>
      <property name="placeholderText">
       <string>Name</string>
      </property>
      <property name="clearButtonEnabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="surname_edit">
      <property name="styleSheet">
       <string notr="true">color: rgb(176, 121, 70);font: 8pt &quot;Eras Demi ITC&quot;; font-size:20pt;</string>
      </property>
      <property name="placeholderText">
       <string>Surname</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="number_edit">
      <property name="styleSheet">
       <string notr="true">color: rgb(176, 121, 70);font: 8pt &quot;Eras Demi ITC&quot;; font-size:20pt;</string>
      </property>
      <property name="placeholderText">
       <string>Number</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="addres_edit">
      <property name="styleSheet">
       <string notr="true">color: rgb(176, 121, 70);font: 8pt &quot;Eras Demi ITC&quot;; font-size:20pt;</string>
      </property>
      <property name="placeholderText">
       <string>Address</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDateEdit" name="datebirth_edit">
      <property name="contextMenuPolicy">
       <enum>Qt::DefaultContextMenu</enum>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(176, 121, 70);font: 8pt &quot;Eras Demi ITC&quot;; font-size:20pt;</string>
      </property>
      <property name="wrapping">
       <bool>false</bool>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
      <property name="specialValueText">
       <string comment="Date of birth" extracomment="Date of birth"/>
      </property>
      <property name="accelerated">
       <bool>true</bool>
      </property>
      <property name="showGroupSeparator" stdset="0">
       <bool>false</bool>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="confirm_but">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(239, 228, 212);
font: 63 8pt &quot;Yu Gothic UI Semibold&quot;;
font-size: 16pt;</string>
        </property>
        <property name="text">
         <string>Ok</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="exit_add_but">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(239, 228, 212);
font: 63 8pt &quot;Yu Gothic UI Semibold&quot;;
font-size: 16pt;</string>
        </property>
        <property name="text">
         <string>Exit</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>431</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sitka Heading</family>
     <pointsize>24</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(21, 22, 24); 
font: 8pt &quot;Sitka Heading&quot;; font-size:24pt;</string>
   </property>
   <property name="text">
    <string>Changed contact number:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="first_num_edit">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>100</y>
     <width>569</width>
     <height>43</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(176, 121, 70);font: 8pt &quot;Eras Demi ITC&quot;; font-size:20pt;</string>
   </property>
   <property name="placeholderText">
    <string>Contact number</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
